src/assets/lima-config.yaml

```diff
@@ -30,3 +30,9 @@ provision:
     done
     echo "Done mounting Rancher persistent volumes."
     exit "${errors}"
+- # This sets up cron (used for logrotate)
+  mode: system
+  script: |
+    #!/bin/sh
+    rc-update add crond default
```

Comment:
Adding `crond` to the `default` runlevel isn't necessary because the provisioning script runs on every boot, so it will always be started manually anyways.

Reply:
I opted to do that in case the user went in and poked about, manually changing runlevels and whatnot.  We _do_ want crond to be running by default, even if we reached that runlevel by the user manually running `openrc default`.